osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -91,7 +91,7 @@ def extract_excerpts(execution_configuration, extraction_order, bbox_args, conve
             index = 0
             if export_format_key in execution_configuration['formats']:
                 index += 1
-                extraction_command = "docker-compose run excerpt python excerpt.py %(bbox_args)s -f %(format)s" % {
+                extraction_command = "docker-compose run --rm excerpt python excerpt.py %(bbox_args)s -f %(format)s" % {  # noqa
```

Comment:
line too long. I tried with line breaking, but it just gets worse from a readability standpoint.